crds/main.go

```diff
@@ -0,0 +1,266 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/pulumi/pulumi/pkg/v2/codegen/nodejs"
+	"github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+	pschema "github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+	unstruct "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const tool = "crdgen"
+
+func main() {
+	yamlPath := os.Args[1]
+	yamlFile, err := ioutil.ReadFile(yamlPath)
+	if err != nil {
+		log.Printf("yamlFile.Get err	#%v ", err)
```

Comment:
Nit:

Once we move to `cobra`, it's slightly nicer as you can [just return the error](https://github.com/pulumi/tf2pulumi/blob/1e7538fb9a348411e030c08ea609722525c5f151/main.go#L58), but in the meantime, we should do something like:

```go
if err != nil {
	fmt.Fprintf(os.Stderr, "error: %v\n", err)
	os.Exit(-1)
}
```

Or if you want to provide more details, you could import `"github.com/pkg/errors"` and use `errors.Wrap`:

```go
if err != nil {
	fmt.Fprintf(os.Stderr, "error: %v\n", errors.Wrap(err, "reading the yaml file"))
	os.Exit(-1)
}
```